### ISSUE TITLE
While browsing a feed, bottom bar buttons cannot be operated with TalkBack

### ISSUE THREAD

**Issue by dankeck** [daniebeler/pixelix#126]
### Testing system

Pixel 5
Android 14
Pixelix v3.1.0

### Steps to reproduce

- Start the TalkBack screen reader.
- Open Pixelix and view the Global feed.
- Attempt to navigate to the bottom bar, either using [Explore by touch or Explore by swiping](https://support.google.com/accessibility/android/answer/6006598?hl=en). 

### Expected behavior

The Home, Search, Trending. Notifications, and profile buttons are read and can be activated.

### Actual behavior

- When using Explore by swiping, the bottom bar buttons are never reached.
- When using Explore by touch, the bottom bar buttons are never focused.

![The bottom bar of the Pixelix Android app, showing 5 icons: a house, magnifying glass, squiggly arrow, heart, and person](https://github.com/user-attachments/assets/421c5012-30da-405c-a58b-a9c6e4fc34cf)

**Comment by Hiebeler** [comment]
This should be fixed with this comment and will be released in the next update.
